prepare-efi.sh

```diff
@@ -14,12 +14,29 @@ elif [ "$ARCH" = "aarch64" ]; then
     PACKAGES=grub2-efi-aa64
     BOOTEFI=BOOTAA64.EFI
     GRUBEFI=grubaa64.efi
+    ARCH=$(echo $ARCH | sed 's/aarch/arm/')
 else
     echo "WARNING: don't know how to build an EFI image on $ARCH"
     touch "$DEST"
     exit 0
 fi
 
+if [ "$OS" = "centos" ]; then
```

Comment:
If you want to have this in a script, let's create `build-ipxe` or something. This is not EFI-specific.

Reply:
you mean a separate script?
my idea is that having all into one script would simplify downstream backport

Reply:
This block creates files that are referenced in Dockerfile causing it to fail if $OS != "centos",
I guess this is ok as downstream it will be used by another Dockerfile? if so can you add a comment to explain

Reply:
This block creates files that are referenced in Dockerfile causing it to fail if $OS != "centos",
I guess this is ok as downstream it will be used by another Dockerfile? if so can you add a comment to explain

Reply:
> my idea is that having all into one script would simplify downstream backport

Then we need to rename this script? Dunno, I'd not mix two unrelated things.